showthread.php

```diff
@@ -120,7 +120,9 @@
 }
 else
 {
-	$ismod = false;
+	$ismod = false;	$ismod = false;
```

Comment:
Duplicate.

Also, this has nothing to do with that issue either, but whatever..